src/components/PathwayCourse/index.js

```diff
@@ -106,36 +146,156 @@ function PathwayCourse() {
-                  <Typography
-                    variant="body2"
-                    className={classes.cardSubtitle}
-                    sx={{ textAlign: isActive && "center", pb: "8px" }}
-                  >
-                    Learning Track
-                  </Typography>
-                  <Typography
-                    variant="h4"
-                    className={classes.heading}
-                    sx={{ textAlign: isActive && "center", pb: "16px" }}
+            <PathwayCards
+              userEnrolledClasses={userEnrolledClasses}
+              data={data}
+            />
+          </>
+        ) : (
+          pathwayId &&
+          pathwayCourseData && (
+            <>
+              <Grid
+                container
+                spacing={2}
+                align="center"
+                className={classes.box}
+              >
+                <Grid item xs={12} md={6} sx={{ pl: 2 }}>
+                  <Card
+                    align="left"
+                    elevation={0}
+                    className={classes.titleCard}
                   >
-                    {pathwayCourseData.pathway}
-                  </Typography>
-                  <Typography variant="body1">
-                    {pathwayCourseData.description}
-                  </Typography>
-                </Card>
-              </Grid>
-              {/* <Grid xs={12} md={6} sx={{ pl: 2 }}>
+                    <Typography
+                      variant="body2"
+                      className={classes.cardSubtitle}
+                      sx={{ textAlign: isActive && "center", pb: "8px" }}
+                    >
+                      Learning Track
+                    </Typography>
+                    <Typography
+                      variant="h4"
+                      className={classes.heading}
+                      sx={{ textAlign: isActive && "center", pb: "16px" }}
+                    >
+                      {pathwayCourseData.pathway}
+                    </Typography>
+                    <Typography variant="body1">
+                      {pathwayCourseData.description}
+                    </Typography>
+
+                    <ExternalLink
+                      style={{
+                        textDecoration: "none",
+                      }}
+                      href={"https://youtu.be/DDFvJmC3J5M"}
+                    >
+                      <Typography
+                        style={{ display: "flex" }}
+                        mt={2}
+                        align="start"
+                        variant="body2"
+                      >
+                        <img
+                          src={require("./asset/ComputerScreen.svg")}
+                          alt="MonitorScreen Img"
+                        />
+                        <section
+                          className={classes.link}
+                          onClick={handleVideo}
```

Comment:
This is not needed as the function just returns a React element, which is not used.  If you wanted to embed the YouTube video instead of link to it, you might use `{handleVideo()}` on line 250, but then you'd also rename the function like `IntroVideo` to indicate it's a component and not a click handler and use something like `<IntroVideo />` instead.